src/wp-includes/navigation-areas.php

```diff
@@ -50,3 +50,202 @@ function get_navigation_areas() {
 	global $navigation_areas;
 	return $navigation_areas;
 }
+
+/**
+ * Migrates classic menus to a block-based navigation post on theme switch.
+ * Assigns the created navigation post to the corresponding navigation area.
+ *
+ * @since 5.9.0
+ * @access private
+ *
+ * @param string   $new_name  Name of the new theme.
+ * @param WP_Theme $new_theme New theme.
+ * @param WP_Theme $old_theme Old theme.
+ */
+function _migrate_menu_to_navigation_post( $new_name, $new_theme, $old_theme ) {
```

Comment:
```suggestion
function _wp_migrate_menu_to_navigation_post( $new_name, $new_theme, $old_theme ) {
```

Let's prefix with wp to careful.